### PR TITLE
16.0 mig project role

### PR DESCRIPTION
**Pull request by uncannycs**

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

File: project_role/models/project_role.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    )
+    child_ids = fields.One2many(
+        string="Child Roles",
+        comodel_name="project.role",
+        inverse_name="parent_id",
+        copy=True,
+    )
+    complete_name = fields.Char(
+        compute="_compute_complete_name",
+        store=True,
```
Comment (CRogos) [inline project_role/models/project_role.py:37]: There was a patch fixing a recursive warning which is missing:
https://github.com/OCA/project/commit/1a07a6c300a6703f8c122084484bdd5405fc66a7

**Review by CRogos — CHANGES_REQUESTED** [review]

File: project_role/models/project_role.py (hunk 1 above)
Comment (leemannd) [reply project_role/models/project_role.py:37]: Hello @kg-ucs , Do you have time to have a look at it?

Comment (leemannd) [reply project_role/models/project_role.py:37]: @CRogos Included into -> https://github.com/OCA/project/pull/1049

**Comment by leemannd** [comment]
Closing it has been superseed by https://github.com/OCA/project/pull/1049

**Pull request closed without merging**